File: javascript/3396.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function (nums) {
    let f = new Array(101).fill(false);
    for (let i = nums.length - 1; i >= 0; i--)
        if (f[nums[i]])
            return parseInt(i / 3) + 1;
        else
            f[nums[i]] = true;
    return 0;
};

File: java/3396.java
class Solution {
    public int minimumOperations(int[] nums) {
        boolean f[] = new boolean[101];
        for (int i = nums.length - 1; i >= 0; i--)
            if (f[nums[i]])
                return i / 3 + 1;
            else
                f[nums[i]] = true;
        return 0;
    }
}
